### PR TITLE
Check in basic keymaster-certmond.

### PR DESCRIPTION
**Pull request by rgooch**

### CODE REVIEW

File: certmon/httpd/status.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	fmt.Fprintln(writer, "<h3>")
+	s.writeDashboard(writer)
+	for _, htmlWriter := range htmlWriters {
+		htmlWriter.WriteHtml(writer)
+	}
+	fmt.Fprintln(writer, "</h3>")
+	fmt.Fprintln(writer, "<hr>")
+	html.WriteFooter(writer)
+	fmt.Fprintln(writer, "</body>")
+}
```
Comment (cviecco) [inline certmon/httpd/status.go]: This should definetively be made into an html template... but that is for another dat

File: certmon/monitord/impl.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+		SshRawCertChannel:  sshRawCertChannel,
+		SshCertChannel:     sshCertChannel,
+		X509RawCertChannel: x509RawCertChannel,
+		X509CertChannel:    x509CertChannel,
+	}
+	go monitor.forever(logger)
+	return monitor, nil
+}
+
+func (m *Monitor) forever(logger log.Logger) {
```
Comment (cviecco) [inline certmon/monitord/impl.go]: forever.... and ever is a great chorus.... but not very descriptive... a better name would be nice... again just mark it a as a TODO.

Comment (rgooch) [reply certmon/monitord/impl.go]: OK, I've renamed to `monitorForever()`.

File: certmon/httpd/status.go (hunk 1 above)
Comment (rgooch) [reply certmon/httpd/status.go]: I've been thinking of making a library function/package for this. I think that's a better fit than a template. There are several *Dominator* servers that could use this.

**Pull request merged**